app/controllers/static_controller.rb

```diff
@@ -103,4 +111,60 @@ def default(input_data, default)
     end
   end
 
+  def seed_data_check()
+    if session[:is_match_data_complete] == nil || !session[:is_match_data_complete]
+      lorem_ipsum = "Lorem ipsum sit amet, consectur adipiscing elit sed do eiusmod tempor inciditunt ut labore et dolore magnum."
+      reason_1 = "Highly compatible products"
+      reason_2 = "High levels of community involvement"
+      reason_3 = "Similar business ideologies"
+
+      session[:possible_match_name] = "Sweet Inspriations" if session[:possible_match_name] == nil
```

Comment:
In all of the places you do `if something == nil`, you can just do `if something`. it is a little cleaner and more readable